idmapping-common/src/main/java/org/uniprot/api/idmapping/common/service/BasicIdService.java

```diff
@@ -8,27 +8,22 @@
 import org.apache.solr.client.solrj.io.stream.TupleStream;
```

Comment:
I think, it is better if we could deploy this branch in dev and do idmapping testing through the uniprot website.
1. Few Idmapping jobs
2. IdMapping with facets and sort
3. just ids etc